lib/local_system.rb

```diff
@@ -29,18 +29,40 @@ def os
       @@os
     end
 
-    def validate_existence_of_package(package)
-      begin
-        Cheetah.run("rpm", "-q", package)
-      rescue
-        needed_module = os.module_required_by_package(package)
-        if needed_module
-          raise(Machinery::Errors::MissingRequirement.new("You need the package '#{package}' from module '#{needed_module}'. You can install it as follows:\n" \
-            "If you haven't selected the module '#{needed_module}' before, run `yast2 scc` and choose 'Select Extensions' and activate '#{needed_module}'.\nRun `zypper install #{package}` to install the package."))
-        else
-          raise(Machinery::Errors::MissingRequirement.new("You need the package '#{package}'. You can install it by running `zypper install #{package}`"))
+    def validate_existence_of_packages(packages)
```

Comment:
Assignment Branch Condition size for validate_existence_of_packages is too high. [16.91/15]